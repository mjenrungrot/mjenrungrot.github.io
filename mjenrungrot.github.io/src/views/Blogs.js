// @flow

// eslint-disable-next-line no-unused-vars
import React from 'react';
// eslint-disable-next-line no-unused-vars
import ReactDOM from 'react-dom';
import PropTypes from 'prop-types';

import {withStyles} from '@material-ui/core/styles';

import Blog from '../components/Blog';
import Main from '../layouts/Main';

const styles = (theme) => ({
  root: {
    ...theme.mixins.gutters(),
    paddingTop: theme.spacing(2),
    paddingBottom: theme.spacing(2),
  },
});

/**
 * Blogs View Component
 * @param {object} props creation params
 * @return {object} Blogs component
 */
function Blogs(props) {
  return (
    <Main selectedTab="/blogs">
      <Blog />
    </Main>
  );
}

Blogs.propTypes = {
  classes: PropTypes.any.isRequired,
};

export default withStyles(styles)(Blogs);
