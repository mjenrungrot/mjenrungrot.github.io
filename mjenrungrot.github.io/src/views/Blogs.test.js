// eslint-disable-next-line no-unused-vars
import React from 'react';
import {shallow} from 'enzyme';
import Blogs from './Blogs';

describe('Blogs component', () => {
  test('renders', () => {
    const component = shallow(<Blogs />);
    expect(component).toMatchSnapshot();
  });
});
