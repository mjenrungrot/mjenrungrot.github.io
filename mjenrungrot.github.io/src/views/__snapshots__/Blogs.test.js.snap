// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blogs component renders 1`] = `
<Blogs
  classes={
    Object {
      "root": "Blogs-root-1",
    }
  }
/>
`;
